text/000-merge-with-rubygems.md

```diff
@@ -0,0 +1,100 @@
+# Merge Bundler and RubyGems
+
+- Feature Name: bundler\_rubygems\_merger
+- Start Date: 2019-04-23
+- RFC PR: (leave this empty)
+- Bundler Issue: (leave this empty)
+
+## Summary
+
+Let's combine Bundler and RubyGems into one thing, with one team, repo, issues, version number, release cycle, and all the other things. When everything is done,  `bundle install` will be a feature provided by RubyGems.
+
+## Motivation
+
+When Bundler was first created, in the late 2000s, it had very different goals than RubyGems had at the time. For several years after Bundler launched, the Bundler and RubyGems teams had different goals, and different ideas about how to reach those goals.
+
+However, over the last ten years Bundler and RubyGems have become more and more overlapping. Today, the Bundler and RubyGems teams are almost all the same people, and many changes to one of Bundler or RubyGems also require changes in the other.
+
+As Bundler and RubyGems continue to overlap more and more, especially now that they both ship inside Ruby itself, it keeps getting more and more annoying to keep them separate.
+
+In order to make future development and releases easier, let's combine Bundler with RubyGems, both organizationally (team membership, GitHub org and issues, git repositories) and technically (combined repo, codebase, version number, releases, and install process).
+
+Combining some things will be easy, although combining everything will take some experimentation, development work, testing, and future releases. It seems worth it.
+
+## Guide-level explanation
+
+** Before the code-level merge:**
+
+Bundler 3 is a stepping stone on the way to combining Bundler and RubyGems. You can install it by running `gem install bundler`. You will notice that there are some changes to how the `bundle` command works.
+
+We've been planning and working towards those changes for a long time, and we think you'll find Bundler easier and more straightforward to use once you've had a chance to get used to the changes.
+
+We've already started working on the next version of Bundler, which will be fully part of RubyGems. It will not need to be installed separately, and will always be available anywhere you have Ruby and RubyGems available.
```

Comment:
> It will not need to be installed separately

can you clarify: can it be updated separately? will ruby gems work without it?

Reply:
> anywhere you have Ruby and RubyGems available

is distro package manager un-bundling a concern here?

Reply:
Assuming we are able to work out the technical/UX concerns (I have a proposal that I will open as a separate RFC soon), it will not be possible to update Bundler separately from RubyGems. 

Distros that break up stdlib always set dependencies on the versions that ship together, so it shouldn't be any more broken than the usual stdlib splitting up already is.